jupyter-org-extensions.el

```diff
@@ -0,0 +1,401 @@
+;;; jupyter-org-extensions.el --- Jupyter Org Extensions -*- lexical-binding: t; -*-
+
+;; Copyright (C) 2018 Nathaniel Nicandro
+
+;; Author: Carlos Garcia C. <carlos@binarycharly.com>
+;; Created: 01 March 2019
+;; Version: 0.7.1
+
+;; This program is free software; you can redistribute it and/or
+;; modify it under the terms of the GNU General Public License as
+;; published by the Free Software Foundation; either version 2, or (at
+;; your option) any later version.
+
+;; This program is distributed in the hope that it will be useful, but
+;; WITHOUT ANY WARRANTY; without even the implied warranty of
+;; MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+;; General Public License for more details.
+
+;; You should have received a copy of the GNU General Public License
+;; along with GNU Emacs; see the file COPYING.  If not, write to the
+;; Free Software Foundation, Inc., 59 Temple Place - Suite 330,
+;; Boston, MA 02111-1307, USA.
+
+;;; Commentary:
+
+;; Functions that extend the functionality of Org mode to interact with
+;; jupyter source blocks.
+
+;;; Code:
+
+(require 'org)
+(require 'jupyter-org-client)
+
+(declare-function org-babel-jupyter-initiate-session "ob-jupyter" (&optional session params))
+(declare-function avy-with nil)
```

Comment:
Can you add the package where these come from, e.g. using `(declare-function avy-with "ext:avy")`